packages/autorest.gotest/test/integrationtest/output/signalr/armsignalr/__debug/go-tester-pre.yaml

```diff
@@ -5064,7 +5064,7 @@ operationGroups:
             Operations_List:
               parameters:
                 api-version: 2021-06-01-preview
-              x-ms-original-file: file:///C:/git/Azure/autorest.go/swagger/specification/signalr/resource-manager/Microsoft.SignalRService/preview/2021-06-01-preview/examples/Operations_List.json
+              x-ms-original-file: file:///D:/w/t/autorest.go/swagger/specification/signalr/resource-manager/Microsoft.SignalRService/preview/2021-06-01-preview/examples/Operations_List.json
```

Comment:
Will this file path impact the pipeline run?

Reply:
nope, but the file is auto-generated. i don't want to change it manually.